bot.py

```diff
@@ -46,8 +47,8 @@ class DiscordBot(BaseBot):
     DEFAULT_PREFIX = '!'
     DEFAULT_LANGUAGE = 'en'
     BOT_NAME = 'garyatrics.com'
-    BASE_GUILD = 'Garyatrics'
-    VERSION = '0.6'
+    BASE_GUILD = "Eric's Test Server"
```

Comment:
this should be reverted. i will later put that into an external configuration file.

Reply:
Fixed